depression-and-anxiety/getting-therapy.md

```diff
@@ -13,14 +13,16 @@ Talking to a therapist can help with:
 - finding ways of coping every day
 
 Usually treatment for depression and anxiety can include: talking to a therapist, medication, and self-help such as exercise.
-
+a
 There are many different types of talking treatments.  The charity Mind has more information about [types of therapy](http://www.mind.org.uk/information-support/drugs-and-treatments/talking-treatments/types-of-therapy/#.VmBFKHbhCM8). 
 
 ##How to get therapy on the NHS
 
 1. Speak to your GP about the way you feel.
-2. If your GP thinks therapy is the best treatment for you they’ll give you the number for a service that provides therapy in your area.
-3. You phone the local service, they’ll ask you some questions and refer you to one of their therapists.
+2. If your GP thinks therapy is the best treatment, they’ll either refer for therapy in your area or give you the number for a service that provides therapy in your area.
```

Comment:
On 7 December 2015 at 12:03, Dominic Baggott <notifications@github.com>
wrote:

> Should this read "refer *you* for therapy"?
>


​yes it should I reckon​